src/main/webapp/app/admin/statistics/statistics.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { SERVER_API_URL } from 'app/app.constants';
+import { SpanType } from 'app/entities/statistics.model';
+
+@Injectable({ providedIn: 'root' })
+export class StatisticsService {
+    private resourceUrl = SERVER_API_URL + 'api/';
```

Comment:
You could probably change this to `'api/management/statistics/'` and adjust the other urls.

Reply:
[Done](https://github.com/ls1intum/Artemis/pull/2471/commits/26c304b14d682a3ce150db83b5a908274b647375)